robottelo/manifests.py

```diff
@@ -166,5 +174,6 @@ def original_manifest():
 
         with original_manifest() as manifest:
             # my fancy stuff
+    :parameter manifest: a key from settings.fake_manifest.urls
     """
-    return Manifest(_manifest_cloner.original())
+    return Manifest(_manifest_cloner.original(manifest))
```

Comment:
Does this work? `_manifest_cloner` is a dict not an object. I think this would be something like `_manifest_cloner[menifest].original()`